### PR TITLE
Adding dims as approver for k8s.io

### PR DESCRIPTION
**Pull request by dims**
@thockin unblocked me from being able to do things with https://github.com/kubernetes/k8s.io/pull/1796

thanks Tim! 

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

File: k8s.io/OWNERS [hunk 1]
```diff
@@ -5,6 +5,7 @@ options:
 approvers:
 - ixdy
 - thockin
+- dims
```
Comment (spiffxp) [inline k8s.io/OWNERS:8]: We should get parity between approvers and this group https://github.com/kubernetes/k8s.io/blob/390481ba911059c23a2739cb56bccf8be3ea183b/groups/groups.yaml#L543-L554

```suggestion
- aojea
- bentheelder
- cblecker
- dims
- spiffxp
```

(you'll also need to remove ixdy)

Can you update this to add yourself to the group? Add thockin and myself (@google.com) if it's not too much trouble?

**Review by spiffxp — CHANGES_REQUESTED** [review]
since I don't have approve, let's make this followup I never got around to today mentioned in https://github.com/kubernetes/k8s.io/pull/1792#discussion_r595494784

I'm also fine if we decide to prune this list of people but I feel like parity first, prune followup

**Comment by dims** [comment]
@spiffxp this is ready now. thanks!

**Review by spiffxp — APPROVED** [review]
/lgtm

Group updates are missing but that can be a followup

**Pull request merged**